src/activity_actor.cpp

```diff
@@ -5123,6 +5137,19 @@ static ret_val<void> try_insert( item_location &holster, drop_location &holstere
     ret_val<int> max_parent_charges = holster.max_charges_by_parent_recursive( it );
     if( !max_parent_charges.success() ) {
         return ret_val<void>::make_failure( max_parent_charges.str() );
+    } else if( !carrier->is_npc() && max_parent_charges.value() < holstered_item.second ) {
+        // if you cannot fit all items because parent container is too small
+        if( query_yn(
+                _( "Only %i items fit into %s due tu the parent container being too small.  Would you like to wield %s to fit more items?" ),
+                max_parent_charges.value(), holster->tname(), holster->tname() )
```

Comment:
Wording is not my strongest suit, but consider something like:
```suggestion
                _( "The parent container does not have enough space to fit more than %s. Would you like to wield %s first to fit more?" ),
                it.tname(max_parent_charges.value()), holster->tname() )
```
(and if you change this one, consider adjusting the other query similarly as well)